.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+#### Expected behavior vs actual behavior
+<br><br>
+
+#### Steps to reproduce
+*(e.g., detailed walkthrough, runnable script, example application)*
+<br><br><br>
+
+#### Environment
+
+AMS Version *(commit ref if not on tag)*: 
```

Comment:
I'd prefer to refer to AMS always as `ActiveModelSerializers` in the docs. Any objections?